patcher/src/ts/sidebar/CharacterSelect.tsx

```diff
@@ -5,6 +5,11 @@
  */
 
 import * as React from 'react';
+import {components, race, restAPI} from 'camelot-unchained';
```

Comment:
https://github.com/airbnb/javascript#whitespace--in-braces

    import { things, things } from 'something';

rather than

    import {things, things} from 'something';